Fix size of memset which is what ctx points to.

--- src/crypto/md5-internal.c	2012-01-18 10:02:40.000000000 +0100
+++ src/crypto/md5-internal.c	2017-01-09 00:57:00.889939319 +0100
@@ -188,7 +188,7 @@
     MD5Transform(ctx->buf, (u32 *) ctx->in);
     byteReverse((unsigned char *) ctx->buf, 4);
     os_memcpy(digest, ctx->buf, 16);
-    os_memset(ctx, 0, sizeof(ctx));	/* In case it's sensitive */
+    os_memset(ctx, 0, sizeof(*ctx));	/* In case it's sensitive */
 }
 
 /* The four core functions - F1 is optimized somewhat */
